{"name": "test_confirm_cancel", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "driver_fixture = <appium.webdriver.webdriver.WebDriver (session=\"698b5393-cd96-47b9-afcf-687d33d49b8a\")>\n\n    @pytest.fixture(scope='session',autouse=False)\n    def login_fixture(driver_fixture):\n        csv_file=BASE_DIR + '\\datas\\login.csv'\n        print(\"csv_file:%s\"%csv_file)\n        driver=driver_fixture\n        com=Common(driver)\n        data=com.get_csv_data(csv_file,1)\n        time.sleep(2)\n        trd=First(driver)\n        trd.to_login()\n        logn=Login(driver)\n>       logn.login(data[0],data[1])\n\nconftest.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\businessPage\\login_page.py:110: in login\n    self.get_visible_element(self.tv_account,\"首页账号\",30)\n..\\common\\common_fun.py:212: in get_visible_element\n    raise e\n..\\common\\common_fun.py:207: in get_visible_element\n    ele=WebDriverWait(self.driver,timeout,0.5).until(EC.visibility_of_element_located(loc))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"698b5393-cd96-47b9-afcf-687d33d49b8a\")>\nmethod = <selenium.webdriver.support.expected_conditions.visibility_of_element_located object at 0x042C3790>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n..\\..\\python\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "attachments": [{"name": "log", "source": "49dc9ab2-2366-46d2-b792-ce36d2c9884d-attachment.txt", "type": "text/plain"}], "start": 1652710646027, "stop": 1652710646027, "uuid": "76d36d2d-fa36-48ba-be41-a4e6a9898904", "historyId": "1f733a2d20c27314b0407d8bfa2df89c", "testCaseId": "d7a1476414b243320b16d6de73c6818d", "fullName": "testCancelOrder.test_Qcancelorder.TestCancelOrder#test_confirm_cancel", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('back_fixture')"}, {"name": "parentSuite", "value": "testCancelOrder"}, {"name": "suite", "value": "test_Qcancelorder"}, {"name": "subSuite", "value": "TestCancelOrder"}, {"name": "host", "value": "Win102021AVUCSJ"}, {"name": "thread", "value": "1400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCancelOrder.test_Qcancelorder"}]}